Sources/SwiftCrossUI/Values/Edge.swift

```diff
@@ -0,0 +1,27 @@
+/// An enumeration to indicate one edge of a rectangle.
+public enum Edge: Int8, CaseIterable, Hashable {
+    case top
+    case bottom
+    case leading
+    case trailing
+
+    /// An efficient set of Edges.
+    public struct Set: OptionSet, Hashable {
+        public let rawValue : Int8
+        public init(rawValue: Int8) {
+            self.rawValue = rawValue
+        }
+        public init(_ e: Edge) {
+            self.rawValue = e.rawValue
```

Comment:
Have you tested this with multiple edges? I have a feeling this should be `self.rawValue = 1 << e.rawValue` so that the individual edges can be distinguished when multiple raw values are combined.

Also, I prefer to use `edge` instead of `e` even when it's obvious what the single-character variable name means; just a style thing

Reply:
I noticed this too and fixed. 

I also usually use full word but I just copied the api from apple: https://developer.apple.com/documentation/swiftui/edge/set/init(_:)-91lwb

will change

Reply:
Ah makes sense 👍 yeah it's fine to change the internal names for parameters to make more sense as long as the label is the same